text/message_login.go

```diff
@@ -61,7 +61,7 @@ func NewInfoLoginLinkMessage(dupIdentifier, provider, newLoginURL string) *Messa
 		ID:   InfoSelfServiceLoginLink,
 		Type: Info,
 		Text: fmt.Sprintf(
-			"Signing in will link your account to %q at provider %q. If you do not wish to link that account, please start a new login flow.",
+			"You tried signing in with %q, but that email is already linked to another account. Please confirm your account by signing in with one of the options below. This will add your account %[1]q at %q as another way to sign in to your account.",
```

Comment:
```suggestion
			"You tried to sign in with %q, but that email is already used. Sign in with one of the options below to add %q at %q as another sign-in method to the existing account.",
```

Reply:
This is similar to one of my first attempts, too. I revised that to the current proposal (using chatgpt).

> but that email is already used

This is not obvious enough, IMO. E.g. used by what? I'd propose, "Used by another account" makes this clear.

> as another sign-in method to the existing account.

I felt "method" might not be clear enough (and is pretty technical in our context). "Another way to sign in to your account" is pretty clear, although it's a bit longer.

```suggestion
			"You tried to sign in with %q, but that email is already used by another account. Sign in to your account with one of the options below to add your account %[1]q at %q as another way to sign in.",
```

WDYT?

Reply:
I find the sentence a bit confusing (`Sign in to your account with one of the options below to add your account %[1]q at %q as another way to sign in`) but I also don't have a suggestion how to improve it. So it's fine by me.